lambdas/es/indexer/index.py

```diff
@@ -34,7 +34,7 @@
 CHUNK_LIMIT_BYTES = 20_000_000
 DOC_LIMIT_BYTES = 2_000
 ELASTIC_TIMEOUT = 30
-MAX_RETRY = 10 # prevent long-running lambdas due to malformed calls
+MAX_RETRY = 4 # prevent long-running lambdas due to malformed calls
```

Comment:
You can drop `RETRY_429` as well, for the same reasons. Even `RETRY_429 = MAX_RETRY`.